Description: fix ruby gem name in Debian
Author: Vincent Danjean <vdanjean@debian.org>
Forwarded: not-needed
---
--- a/kameleon-builder.gemspec
+++ b/kameleon-builder.gemspec
@@ -31,8 +31,8 @@
   s.require_paths = ['lib']
 
   s.add_dependency 'childprocess', '~> 4.0'
-  s.add_dependency 'graphviz', '~> 1.2'
-  s.add_dependency 'progressbar', '~> 1.10'
+  s.add_dependency 'ruby-graphviz', '~> 1.2'
+  s.add_dependency 'ruby-progressbar', '~> 1.10'
   s.add_dependency 'psych', '~> 5.0'
   s.add_dependency 'table_print', '~> 1.5'
   s.add_dependency 'thor', '~> 1.0'
